src/main/java/frc/team2412/robot/subsystem/ShooterSubsystem.java

```diff
@@ -169,6 +173,9 @@ private void configMotors() {
         hoodMotor.setClosedLoopRampRate(1);
         hoodMotor.setIdleMode(CANSparkMax.IdleMode.kBrake);
         setHoodPID(HOOD_DEFAULT_P, HOOD_DEFAULT_I, HOOD_DEFAULT_D, HOOD_DEFAULT_F);
+
+        hoodMotor.enableVoltageCompensation(12.5);
```

Comment:
We should probably have a constant for the voltage, especially since it's used in multiple places.

Reply:
Will fix on the next PR